client/tokenized-express-checkout/blocks/components/express-checkout-button-preview.js

```diff
@@ -0,0 +1,112 @@
+/**
+ * External dependencies
+ */
+import { useEffect, useMemo, useRef } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { getExpressCheckoutButtonAppearance } from 'wcpay/express-checkout/utils';
+
+const ExpressCheckoutButtonPreview = ( {
+	expressPaymentMethod,
+	options,
+	buttonAttributes,
+} ) => {
+	const appearance = useMemo(
+		() => getExpressCheckoutButtonAppearance( buttonAttributes ),
+		[ buttonAttributes ]
+	);
```

Comment:
Similar observation as the one I made in `ExpressCheckoutComponent` - is there a scenario where this `useMemo` would be useful?

Reply:
In this scenario I did it to avoid recalculating the border radius with every render, but the savings are probably negligible, I'll just remove it and see how it performs.